std/machines/memory_with_bootloader_write.asm

```diff
@@ -0,0 +1,81 @@
+use std::array;
+
+/// This machine is a slightly extended version of std::machines::memory::Memory,
```

Comment:
Wasnt this file added in the other PR?

Reply:
Yeah really weird. If you did `diff main` locally, it wouldn't show the diff, but on Github I needed to merge `main` again (which in this case should keep the code unchanged)...